build_chromaprint.bat

```diff
@@ -22,10 +22,20 @@ erase %LIB_DIR%\*chromaprint*
 
 cd build\%CHROMAPRINT_PATH%
 REM Re-generate solution files.
-erase cmakecache.txt
-REM If CMAKE_PLATFORM is empty, CMake generates a Win32 version.
-REM   No separating space is intentional
-cmake . -G "Visual Studio 14 2015%CMAKE_PLATFORM%" -DWITH_FFTW3=ON -DFFTW3_DIR=../..
+rd /S /Q %PLATFORM%
+mkdir %PLATFORM%
+cd %PLATFORM%
+
+REM TODO: verify that the cmake setting BUILD_SHARED_LIBS works correctly.
+SET ADDITIONAL_SETTINGS=
+IF NOT %STATIC_LIBS% (
+SET ADDITIONAL_SETTINGS=-DBUILD_SHARED_LIBS=ON -DWITH_FFTW3_DYNAMIC=ON
+)
+"%CMAKEDIR%\cmake" ../ -G %COMPILERPROJECT% %ADDITIONAL_SETTINGS% -DWITH_FFTW3=ON -DFFTW3_DIR=../../..
```

Comment:
I realize Sean's branch does this too, but FFTW3_DIR should point to %LIBDIR% so we always use the latest built-and-installed FFTW, and not a potentially-dirty build/ path.